### ISSUE TITLE
Add support for Warburg element in circuit_evolution()

### ISSUE THREAD

**Issue by sindresops** [MaximeVH/EquivalentCircuits.jl#21]
Any plans for supporting the "W" as an input argument to the gene expression programing circuit recommendation?

![image](https://user-images.githubusercontent.com/29738855/220930843-d9920153-5ab7-4641-a405-93aac78776b8.png)

**Comment by MaximeVH** [comment]
Note that a Warburg element is a CPE element with its second parameter set to 0.5. If you include the terminals `"RCLP"` and the second parameter of the CPE is close to 0.5, you could consider it as a Warburg element instead.

Because of the above, and because I'm quite busy with other work at the moment, I'm not sure whether I'll include the Warburg element as a separate optional element in the `circuit_evolution` function soon. But I'll add an update on this issue when I do.

**Comment by sindresops** [comment]
I did not actually know that. I will have to think about how that works 🤔 

 But I guess a decision should be made on the whole codebase whether to include "W" or not. The circuit_search() supports it as a separate element, for example. I could have a look too.

**Comment by MaximeVH** [comment]
You're right about that. Also, there may be cases where you'd want to include a Warburg element without including a CPE, which is not possible at the moment.

**Comment by MaximeVH** [comment]
The `circuit_evolution` function now also supports Warburg elements. To include circuits containing such elements, simply include "W" in among the other considered elements in the `terminals` argument.
